src/utils/deriveItemProperties.ts

```diff
@@ -0,0 +1,25 @@
+const Big = require('big.js');
+
+/**
+ * Multiplies an item's weight by its quantity using big.js
+ * Returns the resulting number
+ *
+ * @param {InventoryItemFields} InventoryItemFields The fields for an inventory item
+ * @returns {Number} total item weight
+ */
+export const getItemTotalWeight = (currentItem) => {
```

Comment:
Should type the parameter as `InventoryItemFields` and also add `number` return type